Fauna/Serialization/FaunaType.cs

```diff
@@ -1,21 +1,88 @@
 namespace Fauna.Serialization;
 
+/// <summary>
+/// An enum representing possible <see href="https://docs.fauna.com/fauna/current/reference/fql/types/">Fauna types</see>.
```

Comment:
Clears up a Doxygen warning:
```suggestion
/// An enum representing possible <a href="https://docs.fauna.com/fauna/current/reference/fql/types/">Fauna types</a>.
```